shared/uavobjectdefinition/manualcontrolsettings.xml

```diff
@@ -126,7 +126,7 @@
 
 		<!-- Note these options values should be identical to those defined in FlightMode -->
 		<field name="FlightModeNumber" units="" type="uint8" elements="1" defaultvalue="3"/>
-		<field name="FlightModePosition" units="" type="enum" elements="6" options="Manual,Acro,Leveling,VirtualBar,Stabilized1,Stabilized2,Stabilized3,Autotune,AltitudeHold,VelocityControl,PositionHold,ReturnToHome,PathPlanner,TabletControl" defaultvalue="Manual,Stabilized1,Stabilized2,Stabilized3,ReturnToHome,PositionHold" limits="%0401NE:AltitudeHold:VelocityControl;%0402NE:AltitudeHold:VelocityControl,%0401NE:AltitudeHold:VelocityControl;%0402NE:AltitudeHold:VelocityControl,%0401NE:AltitudeHold:VelocityControl;%0402NE:AltitudeHold:VelocityControl,%0401NE:AltitudeHold:VelocityControl;%0402NE:AltitudeHold:VelocityControl,%0401NE:AltitudeHold:VelocityControl;%0402NE:AltitudeHold:VelocityControl,%0401NE:AltitudeHold:VelocityControl;%0402NE:AltitudeHold:VelocityControl"/>
+		<field name="FlightModePosition" units="" type="enum" elements="6" options="Manual,Acro,Leveling,MWRate,Horizon,AxisLock,VirtualBar,Stabilized1,Stabilized2,Stabilized3,Autotune,AltitudeHold,VelocityControl,PositionHold,ReturnToHome,PathPlanner,TabletControl" defaultvalue="Manual,Stabilized1,Stabilized2,Stabilized3,ReturnToHome,PositionHold" limits="%0401NE:AltitudeHold:VelocityControl;%0402NE:AltitudeHold:VelocityControl,%0401NE:AltitudeHold:VelocityControl;%0402NE:AltitudeHold:VelocityControl,%0401NE:AltitudeHold:VelocityControl;%0402NE:AltitudeHold:VelocityControl,%0401NE:AltitudeHold:VelocityControl;%0402NE:AltitudeHold:VelocityControl,%0401NE:AltitudeHold:VelocityControl;%0402NE:AltitudeHold:VelocityControl,%0401NE:AltitudeHold:VelocityControl;%0402NE:AltitudeHold:VelocityControl"/>
```

Comment:
Their settings in the flash will not be recognized because the object ID will change. 

```
(next)$ cat build/uavobject-synthetics/flight/manualcontrolsettings.h | grep OBJID
#define MANUALCONTROLSETTINGS_OBJID 0x35445246
(flight_modes)$ cat build/uavobject-synthetics/flight/manualcontrolsettings.h | grep OBJID
#define MANUALCONTROLSETTINGS_OBJID 0xA47D63E2
```

If they reimport from a .xml file then it will configure based on the name of the enum and thus work.